src/Microsoft.Azure.SignalR.Common/Utilities/ConnectionStringParser.cs

```diff
@@ -50,7 +50,7 @@ internal static (string endpoint, string accessKey, string version, int? port) P
                 dict.Add(key, kvp[1].Trim());
             }
 
-            if (!dict.ContainsKey(EndpointProperty) || !dict.ContainsKey(AccessKeyProperty))
+            if (!dict.ContainsKey(EndpointProperty))
```

Comment:
is this a breaking change? What happens when user does not enable Aad but missing `accesskey`?